ReleaseChecklist.md

```diff
@@ -2,28 +2,33 @@
 
 ### Requirements
  - [ ] Lauchpad (Ubuntu One) account
- - [ ] gnupg key (has to be version 1, gpg2 won't work) for `your-name@ethereum.org` created and uploaded
- - [ ] Readthedocs account, access to the Solidity project
+ - [ ] gnupg key (has to be version 1, gpg2 won't work) for ``your-name@ethereum.org`` created and uploaded
  - [ ] Write access to https://github.com/ethereum/homebrew-ethereum
 
-### Documentation check
- - [ ] Run `make linkcheck` from within `docs/` and fix any broken links it finds. Ignore false positives caused by `href` anchors and dummy links not meant to work.
-
 ### Blog Post
- - [ ] Create a post on https://github.com/ethereum/solidity-blog and explain some of the new features or concepts.
+ - [ ] Create a post on [solidity-blog](https://github.com/ethereum/solidity-blog) and explain some of the new features or concepts.
+ - [ ] Create a post on [solidity-blog](https://github.com/ethereum/solidity-blog) for each important bug.
+
+### Documentation check
+ - [ ] Run ``make linkcheck`` from within ``docs/`` and fix any broken links it finds. Ignore false positives caused by ``href`` anchors and dummy links not meant to work.
 
 ### Changelog
- - [ ] Sort the changelog entries alphabetically and correct any errors you notice.
- - [ ] Create a commit on a new branch that updates the ``Changelog`` to include a release date.
- - [ ] Run ``./scripts/tests.sh`` to update the bug list.
- - [ ] Create a pull request and wait for the tests, merge it.
+ - [ ] Sort the changelog entries alphabetically and correct any errors you notice. Commit it.
+ - [ ] Update the changelog to include a release date.
+ - [ ] Run ``scripts/update_bugs_by_version.py`` to regenerate ``bugs_by_version.json`` from the changelog and ``bugs.json``.
+       Make sure that the resulting ``bugs_by_version.json`` has a new, empty entry for the new version.
+ - [ ] Commit changes, create a pull request and wait for the tests. Then merge it.
+ - [ ] Copy the changelog into the release blog post.
 
 ### Create the Release
- - [ ] Create Github release page: https://github.com/ethereum/solidity/releases/new
- - [ ] On the release page, select the ``develop`` branch as new target and set tag to the new version (e.g. `v0.8.5`) (make sure you only `SAVE DRAFT` instead of `PUBLISH RELEASE` before the actual release)
+ - [ ] Create a [release on github](https://github.com/ethereum/solidity/releases/new).
+       Set the target to the ``develop`` branch and the tag to the new version, e.g. ``v0.8.5``.
+       Include a warning that the release is still in progress and binaries may not yet be available.
+       Include the following warning: ``**The release is still in progress and the binaries may not yet be available from all sources.**``.
```

Comment:
Yeah - I mean I also just made that up - fine with using anything else, but it's convenient not to have to think about it during the release, but instead to just be able to rely on some template here :-).

Reply:
Oh, just noticed that I left the old line in :) Fixed.

Reply:
I considered actually adding a template of the release notes but decided to leave it out since this is a checklist and should be relatively short. It's probably just as convenient to copy and adjust the one from the last release anyway. It's only the warning that you have to write from scratch every time.